src/main/java/com/softserve/teamproject/service/ScheduleService.java

```diff
@@ -1,29 +1,42 @@
 package com.softserve.teamproject.service;
 
+import com.softserve.teamproject.entity.Event;
 import com.softserve.teamproject.entity.resource.EventResource;
+import java.security.Principal;
 import java.time.LocalDate;
+import java.util.List;
+import javax.xml.bind.ValidationException;
```

Comment:
i think you mentioned this import javax.validation.ValidationException, as in ScheduleServiceImpl.